src/Thumbnail.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import classSet from 'classnames';
+import BootstrapMixin from './BootstrapMixin';
+
+const Thumbnail = React.createClass({
+  mixins: [BootstrapMixin],
+
+  getDefaultProps() {
+    return {
+      bsClass: 'thumbnail'
+    };
+  },
+
+  render() {
+    let classes = this.getBsClassSet();
+
+    if(this.props.href) {
+      return (
+        <a {...this.props} href={this.props.href} className={classSet(this.props.className, classes)}>
```

Comment:
If `this.props.href` is defined and there are children should we warn that the children will not be rendered or render them within this anchor? I'm not sure how Bootstrap does this, or what they prescribe, thoughts?